README.md

```diff
@@ -92,381 +92,167 @@
 <br clear="all" />
 
 ------------------------------------------------------------------------
+# Highlights
 
-Ivy is both an ML transpiler and a framework, currently supporting JAX,
-TensorFlow, PyTorch, and Numpy.
+Ivy is an open-source machine learning framework that
+enables you to:
```

Comment:
support JAX, Machine Learning (ML) libraries such as Tensorflow, Pytorch and Numpy for developing and debugging because of its user friendly usage.